stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/server-config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
just a quick clarification, shouldn't this be 
```
checksum/config
```
instead ? all the charts have used it that way.